UC4-chaincode/src/test/java/de/upb/cs/uc4/chaincode/mock/MockChaincodeStub.java

```diff
@@ -114,7 +114,7 @@ public String getStringState(String key) {
 
     @Override
     public String getFunction() {
-        return null;
+        return txId;
```

Comment:
Please rename variable to "function" or something to be consistent

Reply:
thanks, also noticed I missed the admission contract